src/config.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 //**********************************************************************************************
-//                                       config.php 
+//                                       config.php
```

Comment:
why did you commit the file ? no modification has been done, remove the file update

Reply:
The modification have been done by PHP CS Fixer . It finds an extra space after the filename and remove it.

Reply:
ok for this modification than ;)